corehq/apps/app_manager/south_migrations/__init__.py

```diff
@@ -0,0 +1,203 @@
+import logging
```

Comment:
logging. https://github.com/dimagi/commcare-hq/blob/0f07d0145680a7464568781b2116eba3d57234ae/corehq/apps/app_manager/migrations/__init__.py


I did a bulk `cp -r migrations` `rm -rf migrations/00*.py` so all of the other `__init__.py` show up as blank files.

Looks like this is referenced in a few migrations. I changed those references to point to south_migrations and deleted this content in migrations/__init__.py

(Keeping migrations/__init__.py as it will make making the initial migrations slightly easier for django 1.7)